extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -110,17 +110,17 @@ static public function onCuratedContentSave( $sections ) {
 		WikiaDataAccess::cachePurge( MercuryApiMainPageHandler::curatedContentDataMemcKey() );
 
 		foreach ( $sections as $section ) {
-			if ( !empty( $section['featured'] ) ) {
+			$sectionLabel = $section['label'];
+
+			if ( empty( $sectionLabel ) || !empty( $section['featured'] ) ) {
 				continue;
 			}
 
-			$sectionTitle = $section['title'];
-
-			WikiaDataAccess::cachePurge( MercuryApiMainPageHandler::curatedContentDataMemcKey( $sectionTitle ) );
+			WikiaDataAccess::cachePurge( MercuryApiMainPageHandler::curatedContentDataMemcKey( $sectionLabel ) );
 
 			// Request from browser to MediaWiki
-			$encodedTitle = self::encodeURIQueryParam( $sectionTitle );
-			$urls[] = MercuryApiController::getUrl( 'getCuratedContentSection', [ 'section' => $encodedTitle ] );
+			$encodedTitle = self::encodeURIQueryParam( $sectionLabel );
+			$urls[] = MercuryApiController::getUrl( 'getCuratedContentSection' ) . '&section=' . $encodedTitle;
```

Comment:
`getUrl` method does the additional encoding so I used the same solution as is in https://github.com/Wikia/app/blob/dev/extensions/wikia/CuratedContent/CuratedContentHooks.class.php#L30